pom.xml

```diff
@@ -80,6 +80,16 @@
 			<artifactId>spring-webmvc</artifactId>
 			<version>${spring.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
Given that Fieldbook pom included Middleware artifact, Fieldbook will end having 2 versions for the hibernate-validator artifact (5.2.4.Final added here and 5.1.0.Final added in Middleware) 
Please remove the one in Fieldbook and upgrade the version in Middleware if that the expectation. In case you really need two different versions per project, then exclude the one that comes with Middleware @abatac